### DISCUSSION TITLE
Tokens limit

### DISCUSSION THREAD

**Discussion by FeuFeuAngel** [nod-ai/SHARK#1041] [Q&A]
Is there a way to increase the tokens limit? 

i readed that automatic1111 can bypass that, since i have many prompts, sure its gets slower, but thats fine atm.

**Comment by victordalosto** [comment]
+1
Would appreciate this feature as well.

**Comment by FeuFeuAngel** [comment]
Yeah i got 1111 running with my amd, and node-ai is cutting of the prompts at some point. So you cant make complicated pictures.
